tasks/setup-Debian.yml

```diff
@@ -1,4 +1,45 @@
 ---
+- name: Ensure the WebUpd8 repo key is present.
```

Comment:
@chilcano - I would definitely be willing to add support if it's only minimal changes :)

And if you don't want to maintain a repo role on Ansible Galaxy, I might be able to put one together for you (for the WebUpd8 repo).